lib/chef/provisioning/fog_driver/providers/aws/credentials.rb

```diff
@@ -67,8 +90,13 @@ def load_csv(credentials_csv_file)
 
         def load_default
           config_file = ENV['AWS_CONFIG_FILE'] || File.expand_path('~/.aws/config')
+          credentials_file = ENV['AWS_CREDENTIAL_FILE'] || File.expand_path('~/.aws/credentials')
```

Comment:
Only question I have in here is, should we honor credentials_file over config_file if credentials is present, or vice versa?  Not enough to stop a merge, but worth thinking about.